Medical try 3

Albens opened the pull request:
:cl:
maptweak: Medical has been remapped.
maptweak: Robotics has been remapped.
maptweak: Counselor's office is now where Sci breakroom and Old D1 Checkpoint use to be.
maptweak: Deck 2 maintenance, Starboard midship to Aft has been changed.
maptweak: Bridge Saferoom has been updated to be in line with the new THERA Safe Room.
maptweak: Sci now has a Paperwork office to account for their lost breakroom.
/:cl:

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation
-->

Albens commented:
ah fucking conflict

Albens commented:
#28599 but hopefully functional

Spookerton commented on maps/torch/torch_areas.dm:
```diff
@@ -1498,12 +1500,16 @@
 
-/area/assembly/robotics
-	name = "\improper Robotics Lab"
+/area/assembly/robotics/
```
```suggestion
/area/assembly/robotics
```

Spookerton commented on maps/torch/torch_areas.dm:
```diff
@@ -1076,30 +1070,40 @@
-/area/medical/infirmary
-	name = "\improper Infirmary Hallway"
+/area/medical/medicalhallway
+	name = "\improper Medical Pulic Hallway"
```
```suggestion
	name = "\improper Medical Public Hallway"
```

Albens committed:
Medical try 3

:cl:
maptweak: Medical has been remapped.
maptweak: Robotics has been remapped.
maptweak: Counselor's office is now where Sci breakroom and Old D1 Checkpoint use to be.
maptweak: Deck 2 maintenance, Starboard midship to Aft has been changed.
maptweak: Bridge Saferoom has been updated to be in line with the new THERA Safe Room.
maptweak: Sci now has a Paperwork office to account for their lost breakroom.
/:cl:

Albens commented:
Updated and squished
